internal/flink/command_compute_pool_use.go

```diff
@@ -19,7 +19,6 @@ func (c *command) newComputePoolUseCommand() *cobra.Command {
 		RunE:              c.computePoolUse,
 	}
 
-	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
```

Comment:
Can you add a check to verify that there is such a compute pool in this env and make it fail if not and print an error instructing people to do `environment use`

Reply:
Currently it will tell you this when you try to select a pool from a different env:
```
Error: Flink compute pool "lfcp-123" not found or access forbidden.: Forbidden Access: 403 Forbidden

Suggestions:
    List available Flink compute pools with `confluent flink compute-pool list`
```
I'll add a suggestion to check if the user is in the correct env

Reply:
How's that?
```
Error: Flink compute pool "lfcp-devccnp8rgd" not found or access forbidden.: Forbidden Access: 403 Forbidden

Suggestions:
    List available Flink compute pools with `confluent flink compute-pool list` and make sure you have selected the compute pool's environment with `confluent environment use env-123`

```

Reply:
remove the placeholder env, otherwise good!

Reply:
alright, sounds good!

Reply:
Done